README.md

```diff
@@ -21,3 +21,26 @@ Presentation
 Report
 ---
 #### 
+
+# Documentation
+
+Getting started - Linux
+---
+####
+These steps will work for any Linux distribution with systemd.
+1. Install Docker Desktop, Earthly, and jupyter lab.
+2. In your system terminal, run `sudo usermod -a -G docker $USER`, then reboot.
+3. run `sudo systemctl restart docker`, then run `docker ps` to verify that docker is working. You should see a list in all caps starting with "CONTAINER ID".
+4. run `earthly +docker && ./run-notebook.sh` in the main branch of the project directory.
+5. This will start a length installation process (about 20 minutes). At the end, you may be prompted to reboot. If so, repeat step 3 and run `./run-notebook.sh` in the project directory again.
```

Comment:
Shouldn't be lengthy anymore without the earthly step, and shouldn't need to reboot

Reply:
Updated in #15